### PR TITLE
Feature/remove bounding box

### LINKED ISSUE #68: Remove bounding box thing from the generated svg/ folder
#### Describe the desired behavior

the invisible 24*24 bounding box is not necessary -- see my comments in the following PR: 

>> Be sure to include a 24x24px rectangle with no stroke and no fill to serve as the bounding box.

> I personally did not follow this last time when I was making changes in this repo. (1) we had to manually remove the bounding box when we import them into Figma; (2) the same functionality can be achieved by checking "use artboards" option when the saving/file naming dialog box pop up (so not part of the export options window, which appears in the next screen)
> _Originally posted by @huayunh in https://github.com/pxblue/icons/pull/66#discussion_r461677173_

#### Describe the current behavior

> I checked all of the icons: 109/139 have the bounding box. If we're going to change it, we should make them all the same. We can do that in a separate issue. 
--- Joe

### PR DESCRIPTION
**Pull request by huayunh**
Fixes #68 

#### Changes proposed in this Pull Request:
- Added a nodejs script that can remove all the fill=none 24x24 rectangle

#### Screenshots:

Test in icons-mui repository looks good: 
image

### CODE REVIEW

**Review by joebochill — COMMENTED** [review]
Did we confirm is this is safe to do with respect to the resulting clickable area? See my comment on the original bug.

**Comment by joebochill** [comment]
Looks like the clickable areas are affected by removing the bounding box.

What are the primary motivators for taking it out?

**Comment by huayunh** [comment]
See the bug thread

**Comment by joebochill** [comment]
To be fair, the use case of using an icon like that is pretty low. You can always wrap it in a `<Touchable___>` and you'll be able to click anywhere, which is how we do it in the Header, etc.

Since some of the icons are already without the rect, it's probably safe to say it hasn't been an issue. I'm okay with re-opening this, just wanted us to be aware of the implication.

File: design/Internal/cardsorting.svg [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-      <rect width="24" height="24" fill="none"/>
+      
     </clipPath>
   </defs>
   <title>cardsorting</title>
   <g clip-path="url(#a)">
-    <rect width="24" height="24" fill="none" stroke="#000" stroke-width="0" opacity="0.01"/>
+    
   </g>
   <g>
```
Comment (joebochill) [inline design/Internal/cardsorting.svg:11]: Can we get rid of these empty groups now? Was that clip path actually useful (is there overflow in any of the other paths or anything like that)?

File: scripts/removeBoundingBox.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// from the root folder, run `node ./scripts/removeBoundingBox.js`
+
```
Comment (joebochill) [inline scripts/removeBoundingBox.js:2]: Can we add this as a script in package.json and automatically run it after we create the /svg folder?

**Comment by joebochill** [comment]
@huayunh this has been open for a while. Are we planning to address the changes or would we rather close this and revisit it later?

**Comment by huayunh** [comment]
Personally I am more inclined to close it... that RN thing is really hurting my brain. But still leave the branch around.

**Pull request closed without merging**